core/scripts/chaincli/handler/handler.go

```diff
@@ -47,6 +47,33 @@ const (
 	csaKeysEndpoint              = "/v2/keys/csa"
 )
 
+const (
```

Comment:
I don't expect you to change this as part of this PR (I think I'm just spitballing this idea and I'm happy to make changes myself!)

But I was wondering if it made sense to use this library for generating toml configs:

https://github.com/BurntSushi/toml

So we could define everything in structs and use `toml` tags for the fields, and then use a toml encoder to generate the actual toml files?